### PR TITLE
remove crosswalk to finish #7

### PR DESCRIPTION
**Pull request by dkanada**

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -3,10 +3,10 @@
     <name>Emby</name>
     <description>
     Emby Mobile
-  </description>
+    </description>
     <author email="apps@emby.media" href="http://emby.media">
```
Comment (JustAMan) [inline config.xml]: I guess this has to be changed to our owl mailing list... and URL should be changed as well.

File: platforms/android/res/xml/config.xml [hunk 2]
```diff
@@ -6,10 +6,10 @@
     <name>Emby</name>
     <description>
     Emby Mobile
-  </description>
+    </description>
     <author email="apps@emby.media" href="http://emby.media">
```
Comment (JustAMan) [inline platforms/android/res/xml/config.xml]: same about mailing list and url

File: platforms/android/src/com/mb/android/MainActivity.java [hunk 3]
```diff
@@ -63,17 +62,14 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 import org.apache.cordova.CordovaActivity;
 import com.mb.android.preferences.PreferencesProvider;
-import com.mb.android.webviews.CrosswalkWebView;
 import com.mb.android.webviews.IWebView;
```
Comment (JustAMan) [inline platforms/android/src/com/mb/android/MainActivity.java]: is it planned to get rid of `com.mb.*` things eventually?..

Comment (thornbill) [reply platforms/android/src/com/mb/android/MainActivity.java]: Everything in `/platforms` should be removed from the repository eventually. Any custom functionality that is required should be in a plugin. (See #6)

Comment (JustAMan) [reply platforms/android/src/com/mb/android/MainActivity.java]: I like that new namespace :)

File: config.xml (hunk 1 above)
Comment (thornbill) [reply config.xml]: I had inquired about the email address in jellyfin-dev on Riot and there was talk about adding an `apps@` mailing list for Jellyfin. I don't think that has happened yet though.

Comment (joshuaboniface) [reply config.xml]: @thornbill `apps@jellyfin.org` now exists!

File: platforms/android/res/xml/config.xml (hunk 2 above)
Comment (joshuaboniface) [reply platforms/android/res/xml/config.xml]: `apps@jellyfin.org` now exists.

**Pull request merged**